docs/content/reference/install-configuration/api-dashboard.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Traefik API & Dashboard Documentation"
+description: "Traefik Proxy exposes information through API handlers and showcase them on the Dashboard. Learn about the security, configuration, and endpoints of the APIs and Dashboard. Read the technical documentation."
+---
+
+The dashboard is the central place that shows you the current active routes handled by Traefik.
+
+<figure>
+    <img src="../../../assets/img/webui-dashboard.png" alt="Dashboard - Providers" />
+    <figcaption>The dashboard in action</figcaption>
+</figure>
+
+## Configuration Options
+
+The API and the dashboard can be configured:
+
+- In the Helm Chart: You can find the options to customize the Traefik installation
+enabing the dashboard [here](https://github.com/traefik/traefik-helm-chart/blob/master/traefik/values.yaml#L155).
+- In the Traefik Static Configuration as described below.
+
+| Field      | Description  | Default | Required |
+|:-----------|:---------------------------------|:--------|:---------|
+| `api` | Enable api/dashboard. If the option `api.dashboard` is set to true, this option is by default set to `true` too and should not be explicitly set to `false`. | false     | No      |
+| `api.dashboard` | Enable dashboard | false      | No      |
+| `api.debug` | Enable additional endpoints for debugging and profiling. | false      | No      |
+| `api.disabledashboardad` | Disable the advertisement from the dashboard. | false      | No      |
+| `api.insecure` | Enable the API and the dashboard on the entryPoint named traefik.| false      | No      |
+
+## Configuration Example
+
+Enable the dashboard:
+
+```yaml
+ingressRoute:
+  dashboard:
+    # Enable the dashboard
+    enabled: true
+    # Expose the dashboard on the outside
+    entryPoints: ["websecure"]
+    # Attach the Basic Authentication middleware
+    middlewares: ["dashboard-auth"]
```

Comment:
Here the configuration is the Helm one, maybe you could reuse the example in https://doc.traefik.io/traefik/v2.11/operations/dashboard/#secure-mode.